libs/pilight/protocols/433.92/teknihall.c

```diff
@@ -77,7 +77,7 @@ static void parseCode(void) {
 
 	id = binToDecRev(binary, 0, 7);
 	battery = binary[8];
-	temperature = binToDecRev(binary, 14, 23);
+	temperature = binToSignedRev(binary, 14, 23);
```

Comment:
This will not work. As discussed we need to include bit 13: `temperature = binToSignedRev(binary, 13, 23);`